### PR TITLE
Update README to reflect namespace changes

### PR DESCRIPTION
**Pull request by jleft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,19 +27,18 @@ If you want a quick verification that everything has installed correctly, the fo
 <script type="text/javascript">
 var data = fc.data.random.financial()(50);
```
Comment (jleft) [inline README.md]: N.B. With the current release, this won't work, as it's still `fc.dataGenerator()`. When 4897b48baf942255e6e3bea52060efa0486bb140 is included in the next release, `fc.data.random.financial()` will be correct.

**Comment by chrisprice** [comment]
Ah good spot! Thanks @jleft, we're planning a weekly Friday afternoon release cycle so will merge this tomorrow.

**Comment by jleft** [comment]
No problem. Cool, sounds good :smiley:

**Comment by chrisprice** [comment]
Friday...Monday... who's counting!

**Comment by jleft** [comment]
Haha :+1: thank you :-)

**Pull request merged**